dataflow-website/recipes/scaling/kubernetes/helm/prometheus/prometheus-configmap.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-release-prometheus-server
+  labels:
+    app: prometheus
+data:
+  alert.rules.yml: |-
+    groups:
+      - name: scdfrules
+        rules:
+          - alert: HighThroughputDifference
+            expr: avg(irate(spring_integration_send_seconds_count{name!="errorChannel",name!="nullChannel",result="success",type="channel",application_name="time"}[1m])) by(stream_name)  - avg(irate(spring_integration_send_seconds_count{name!="errorChannel",name!="nullChannel",result="success",type="channel",application_name="transform"}[1m])) by(stream_name) > 500
+            for: 30s
+            annotations:
+              summary: "Throughput delta displace for {{ $labels.stream_name }}"
+              description: "The load-generator throughput is larger than the transform throughput for {{ $labels.stream_name }}"
+          - alert: ZeroThroughputDifference
+            expr: avg(irate(spring_integration_send_seconds_count{name!="errorChannel",name!="nullChannel",result="success",type="channel",application_name="time"}[1m])) by(stream_name)  - 3 * (avg(irate(spring_integration_send_seconds_count{name!="errorChannel",name!="nullChannel",result="success",type="channel",application_name="transform"}[1m])) by(stream_name)) <= 1
+            for: 3m
+            annotations:
+              summary: "Throughput delta for {{ $labels.stream_name }} is zero"
+              description: "The load-generator throughput matches the tranformer throughput for {{ $labels.stream_name }}"
+
+  prometheus.yml: |-
```

Comment:
There's a lot of data elements in this YAML. I assume this was copy + pasted from one of the standard ways folks extend Prometheus config?

Reply:
Yes, unfortunately i had to copy the original configuration to be able to extend it.  We can not modify the content of the `prometheus.yml` yml. Only replace or remove it.  
Discussed this with @chrisjs but could not find a better way.